README.md

```diff
@@ -27,24 +29,29 @@ pip install -e .
 
 ## Usage
 
-### General timelapse-processing
-
-Data to be used for this plugin is typically of the form `[TZYX]` (e.g., 3D + time). Napari-stress offers some convenient way to extent other function's functionality (which are often made for 3D data) to timelapse data using the `frame_by_frame` function and the `TimelapseConverter` class, both of which are described in more detail in [this notebook]([url](https://github.com/BiAPoL/napari-stress/blob/add-timelapse-decorator-for-points-and-surfaces/docs/notebooks/demo/TimeLapse_processing.ipynb)).
+Functionality in this repository is divided in two groups: **Recipes** and **plugins**.
 
 ### Recipes
 
-Napari-stress provides jupyter notebooks with complete workflows for different types of input data. Napari-stress currently provides notebooks for the following data/image types:
+Napari-stress provides jupyter notebooks with [complete workflows](./docs/notebooks/recipes) for different types of input data and objectives. Napari-stress currently provides notebooks for the following data/image types:
+
+* **Confocal data** (*.tif*), 3D+t: This type of data can be processed with napari-stressed as show in [this notebook](./docs/notebooks/recipes/Process_confocal.ipynb)
+
+![](./docs/imgs/confocal/workflow.png)
+
+
+* **Lightsheet data** (*.czi*), 3D + t: coming soon....

```

Comment:
If it's coming soon, remove it from the readme and create a github issue instead.